packages/caret/src/focus.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Set focus to contenteditable or native input element
```

Comment:
> or native input element

this case is not handled

Reply:
why? as i can see any passed element would be focused